addons/binding/org.openhab.binding.gpstracker/README.md

```diff
@@ -24,52 +21,34 @@ Install [OwnTracks for Android](https://play.google.com/store/apps/details?id=or
 Go to Preferences/Connection and set:
 
 * **Mode** - select Private HTTP
-* **Host** - **https://<your.ip.address>/gpstracker/owntracks** or **https://home.myopenhab.org/gpstracker/owntracks**
+* **Host**
+  * https://<your.ip.address>/gpstracker/owntracks or
+  * https://home.myopenhab.org/gpstracker/owntracks
 * **Identification**
   * Turn Authentication ON
-  * Set username and password to be able to reach your openHAB server (myopenhab.org credential, if choosen as URL)
+  * Set username and password to be able to reach your openHAB server (myopenhab.org credential, if choosen as host)
   * Device ID is not important. Set it to e.g. phone
-  * Tracker ID - This id identifies the tracker as a thing. I use initials here.
+  * Tracker ID - This id identifies the tracker as a thing. This must be unique for each tracker connected to the same openHAB instance (e.g. family members).
 
 ![Image](doc/owntracks_setup.png)
 
 ### GPSLogger
 
 Install [GPSLogger for Android](https://play.google.com/store/apps/details?id=com.mendhak.gpslogger) on your device.
 After the launch, go to General Options.
-Enable Start on boot-up and Start on app launch.
+Enable **Start on boot-up** and **Start on app launch**.
 
-![Image](doc/gpslogger_1.png)
+Go to *Logging details* and enable **Log to custom URL**. If you only want to use GPSLogger for this binding, you can disable all other "*Log to*" entries. Right after enabling, the app takes you to the *Log to custom URL* settings:
```

Comment:
If image references are removed pls remove the image resources as well.

Reply:
done